src/lib/Ska/Starcheck/FigureOfMerit.pm

```diff
@@ -59,6 +59,13 @@ sub make_figure_of_merit{
             my $color = $c->{"GS_BV$i"};
             my $hw = $c->{"HALFW$i"};
             my $star_prob = _acq_success_prob($date, $t_ccd, $mag, $color, $spoiler, $hw);
+            if (grep(/Bad Acquisition Star/i, @warnings)){
```

Comment:
Here's one idea for probability overloading.  This uses the same @warnings crap, but those warnings would now be based on the mica acq stats table (though in this split configuration with guide stats still coming from Sybase, both are still needed or the code just defaults back to the rdb files).